src/man/sssd.conf.5.xml

```diff
@@ -769,6 +769,14 @@
                         <para>
                             A value of 0 disables the incrementing behaviour.
                         </para>
+                        <para>
+                            Value of this parameter should be set in corelation
+                            to offline_timeout parameter value.
+                            With offline_timeout set to 60 (default value) there is no point
+                            in setting offlinet_timeout_max to less than 120 as it will
+                            saturate instantly. General rule here should be to set
+                            offline_timeout_max to at least 4 * offline_timeout.
```

Comment:
Here it would be nice to follow the same format, either natural language:
`offline_timeout_max to at least 4 times offline_timeout`
or mathematical:
`offline_timeout_max > 4 * offline_timeout`

Reply:
I chose to follow natural language as it fits better in this description.